index.js

```diff
@@ -93,7 +96,9 @@ document.querySelector('#book-collection').addEventListener('click', (e) => {
   UI.deleteBook(e.target);
 
   // Remove book from Store
-  Storage.removeBook(e.target.previousElementSibling.previousElementSibling.textContent);
+  Storage.removeBook(
+    e.target.previousElementSibling.previousElementSibling.textContent,
+  );
 });
 
-/* eslint-disable max-classes-per-file */
\ No newline at end of file
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] Great job working on this milestone. You are not suppose to disable eslint, rather fix the errors resulting from running the linter checks.